repo/packages/P/prometheus/0/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "prometheus",
+  "version": "0.15.1-0.1",
+  "scm": "https://github.com/prometheus/prometheus",
+  "maintainer": "support@mesosphere.io",
```

Comment:
What should I put there? I understand that those packages aren't supported by mesosphere, but they are still mesosphere packages..